src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -199,6 +199,11 @@ let PostThreadItemLoaded = ({
     return makeProfileLink(post.author, 'post', urip.rkey, 'reposted-by')
   }, [post.uri, post.author])
   const repostsTitle = _(msg`Reposts of this post`)
+  const quotesHref = React.useMemo(() => {
+    const urip = new AtUri(post.uri)
+    return makeProfileLink(post.author, 'post', urip.rkey, 'quotes')
+  }, [post.uri, post.author])
```

Comment:
We really ought to simplify this down to like `uriToRkey` or something. I run into this all the time and hate having to make an `AtUri` every time 🙃. Especially since we could just throw it right into the prop now w/ compiler.

I.e. `makeProfileLink(post.author, 'post', getRkey(uri), 'quotes')`

Minor thing though don't have to do it rn.